cpp/test/Slice/print/AllTests.cpp

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) ZeroC, Inc.
+
+#include "Ice/Ice.h"
+#include "Test.h"
+#include "TestHelper.h"
+
+#include <iomanip>
+
+using namespace std;
+using namespace Test;
+
+void
+Test::Neighbor::ice_print(ostream& os) const
```

Comment:
Example of custom class printing. For custom struct printing, see Ice::Identity, Ice::EncodingVersion.